db/migrate/20200907004651_create_attendances.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAttendances < ActiveRecord::Migration[6.0]
+  def change
+    create_table :attendances, id: :uuid do |t|
+      t.references :child_case_cycle, type: :uuid, null: false, foreign_key: true
+
```

Comment:
We missed two fields in the data model; `check_in_time` and `check_out_time` - waiting for confirmation from Chelsea if that's the right name for the fields, but they should both be `TIME` type fields, formatted only to the minute.  We have some open questions so I think we should keep this PR on hold until we get them answered.

Open questions:

- Is check_in_time and check_out_time the right way to do it?
- Also are those times rounded for calculation of length of care?  Or are they to-the-minute?

Reply:
This model should have `starts_on`, `check_in`, `check_out`, `total_time_in_care`, `slug`, and let's rename `length_of_care` to `attendance_duration`.  We'll do the rounding/logic in a later ticket.

Reply:
Interesting:  https://www.postgresqltutorial.com/postgresql-interval/  I've never used this data type before.  This might suit our needs for `total_time_in_care`?

Reply:
Cool.  postgres `interval` + Ruby `Duration` = yes, that looks like it'd work!